src/vue/forms/PreIssuanceForm.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <form
+    class="app__form issuance-form"
```

Comment:
issuance and pre-issuance are not the same things. do not reuse the assets of each other

Reply:
Resolved, renamed